PyInstaller/utils/hooks/gi.py

```diff
@@ -161,8 +162,12 @@ def gir_library_path_fix(path):
                 f.write(line)
 
         # g-ir-compiler expects a file so we cannot just pipe the fixed file to it.
-        command = subprocess.Popen(('g-ir-compiler', os.path.join(CONF['workpath'], gir_name),
-                                    '-o', os.path.join(CONF['workpath'], typelib_name)))
+        command = subprocess.Popen(
+            (
+                'g-ir-compiler', os.path.join(CONF['workpath'],
+                                              gir_name), '-o', os.path.join(CONF['workpath'], typelib_name)
```

Comment:
This looks weird; I'd prefer if `os.path.join(CONF['workpath'], gir_name)` remained together.